### PR TITLE
Add more resource types

### PR DESCRIPTION
**Pull request by knu**

### CODE REVIEW

**Comment by asok** [comment]
Thanks for your contribution. But I don't feel like putting this package on steroids in order to support the rails plugins. You know you have to write tests for that and later maintain the feature. And since is not rails core it's harder to track changes.

But what I would gladly merge in would be the support for the new rails resources (like ActiveJob's jobs and ActiveModel' validators). Can you make a PR only with such change?
Also for other resource types it could be useful to add a section in the README with an example how user can write his/hers own command to look for Grape's or ActiveAdmin's files.

**Comment by knu** [comment]
@asok I updated this PR as you suggested except for the additional documentation part.

**Comment by asok** [comment]
@knu thanks a lot. I'll look at the failing tests in Travis. Once I find out what's causing them I'll merge your change.

File: projectile-rails.el [hunk 1]
```diff
@@ -518,22 +532,29 @@ The bound variable is \"filename\"."
                                           "/[0-9]\\{14\\}.*_\\(${plural}\\|${singular}\\).*\\.rb$"
                                           'projectile-rails-find-migration))
 
+(defcustom projectile-rails-resource-name-re-list
+  `("/app/models/\\(?:.+/\\)?\\(.+\\)\\.rb\\'"
```
Comment (asok) [inline projectile-rails.el]: @knu what's the idea behind the quote char `'` at the end of re?

Comment (knu) [inline projectile-rails.el]: It is the paranoid version of `$`.  `\'` only matches at the end of a whole string whie `$` also matches before a newline character, if any.

Comment (asok) [inline projectile-rails.el]: Got it, thanks.

Comment (totallymike) [inline projectile-rails.el]: I'm really glad I ran into this question.  I've been curious about exactly what that ending `\\'` means for ages.

**Pull request merged**